english/froboznia/froboznia.md

```diff
@@ -6,4 +6,7 @@ You deftly spin around the clumsy officer's hand and rush out the doorway on the
 other side of the room.  You enter a huge, poorly lit cavern and desperately search
 for an escape route.  You see a rickety bridge spanning a bottomless chasm.
 
-[Take the Bridge] (../get-away/get-away.md)
```

Comment:
Unfortunately, pull requests that delete or modify lines are not eligible for automatic merging.  If you wish to have this pull request auto-merged, please update it to only add lines.  If you don't think you've modified any lines, remember that I consider changing the whitespace (e.g. deleting a space at the end of a line) a modification, even though it may be hard to tell the difference on GitHub.